src/pages/trading-specification/components/_available-platform.tsx

```diff
@@ -102,21 +101,6 @@ const AvailablePlatform = ({ selected_market }: AvailablePlatformProps) => {
                                         </StyledHeader>
                                     </Flex>
                                 </StyledLocalizedLink>
-                                {!selectedEtfsTab ? (
-                                    <StyledLocalizedLink to="/deriv-ez/">
-                                        <Flex direction="row">
-                                            <img
-                                                src={DerivEZ}
-                                                alt={localize('_t_Deriv EZ_t_')}
-                                                width="24"
-                                                height="24"
-                                            />
-                                            <StyledHeader type="paragraph-1" as="h2" width="auto">
-                                                <Localize translate_text={deriv_ez} />
-                                            </StyledHeader>
-                                        </Flex>
-                                    </StyledLocalizedLink>
-                                ) : null}
```

Comment:
Hi , Lets remove line [65](https://github.com/binary-com/deriv-com/pull/5762/files#diff-d5e82e3bcd9954e86a254b94cf57db51274020e3da50925949d6dd4de9156566L65), [68](https://github.com/binary-com/deriv-com/pull/5762/files#diff-d5e82e3bcd9954e86a254b94cf57db51274020e3da50925949d6dd4de9156566L69) as well  , if it's not needed @mayuran-deriv 🫡